spec/granite/form/model/associations/embeds_many_spec.rb

```diff
@@ -392,39 +426,48 @@
     specify { expect(existing_association.concat).to eq(existing_user.projects) }
 
     specify { expect(association.concat(new_project1)).to eq([new_project1]) }
+
     specify do
       expect { association.concat(new_project1) }
-        .to change { association.reader.map(&:attributes) }.from([]).to([{'title' => 'Project 1'}])
+        .to change { association.reader.map(&:attributes) }.from([]).to([{ 'title' => 'Project 1' }])
     end
+
     specify do
       expect { association.concat(new_project1) }
-        .not_to change { user.read_attribute(:projects) }
+        .not_to(change { user.read_attribute(:projects) })
     end
 
     specify do
       expect { existing_association.concat(new_project1, invalid_project) }
         .to change { existing_association.reader.map(&:attributes) }
-        .from([{'title' => 'Genesis'}]).to([{'title' => 'Genesis'}, {'title' => 'Project 1'}, {'title' => nil}])
+        .from([{ 'title' => 'Genesis' }]).to([{ 'title' => 'Genesis' }, { 'title' => 'Project 1' }, { 'title' => nil }])
     end
 
     specify do
       expect { existing_association.concat(new_project1, Dummy.new, new_project2) }
         .to raise_error Granite::Form::AssociationTypeMismatch
     end
+
     specify do
-      expect { muffle(Granite::Form::AssociationTypeMismatch) { existing_association.concat(new_project1, Dummy.new, new_project2) } }
+      expect do
+        muffle(Granite::Form::AssociationTypeMismatch) do
+          existing_association.concat(new_project1, Dummy.new, new_project2)
+        end
+      end
         .to change { existing_association.reader.map(&:attributes) }
-        .from([{'title' => 'Genesis'}]).to([{'title' => 'Genesis'}, {'title' => 'Project 1'}])
+        .from([{ 'title' => 'Genesis' }]).to([{ 'title' => 'Genesis' }, { 'title' => 'Project 1' }])
```

Comment:
It's hard to read, could it be combined with the previous one as `.to raise_error ... .and(not_to(change ...))`?

Reply:
Improved :+1: